src/utils/media-utils.ts

```diff
@@ -0,0 +1,80 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import { getAccessToken } from "@root/utils/token-retrieval-utils"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+
+type ItemType = "dir" | "file"
+
+export interface MediaFile {
```

Comment:
i think we should separate our utils from our types - it's not intuitive for devs to find the type defs for inside a util file and leads to duplication or worse, inaccurate type defs. 

we should shift this out into the `types` folder if they are going to be exported or remove the export otherwise

Reply:
Yeah that makes sense, removed in https://github.com/isomerpages/isomercms-backend/pull/740/commits/6fb5f790b5ed342ad0411fb89e4bfaf590ff4eb1